### PR TITLE
optimize:When the Seata client and server adopt Nacos as the registry, the client will subscribe to the Seata server and the application name will be unknown

### PR DESCRIPTION
**Pull request by hadoop835**
When the Seata client and server adopt Nacos as the registry, the client will subscribe to the Seata server and the application name will be unknown.
1646721213617_1DA5E72A-349A-45d3-8025-434339207EC0

client config
image
image

### CODE REVIEW

**Review by a364176773 — CHANGES_REQUESTED** [review]
script文件夹中client中的相关配置补充一下

**Comment by a364176773** [comment]
标题英文,optimize: 开头

File: seata-spring-autoconfigure/seata-spring-autoconfigure-core/src/main/java/io/seata/spring/boot/autoconfigure/properties/registry/RegistryNacosProperties.java [hunk 1]
```diff
@@ -34,9 +34,10 @@ public class RegistryNacosProperties {
     private String password;
     private String accessKey;
     private String secretKey;
-    private String application = "seata-server";
-    private String slbPattern;
-    private String contextPath;
+    private String application = "seata-server"; 
+    private String clientApplication = "";
```
Comment (a364176773) [inline seata-spring-autoconfigure/seata-spring-autoconfigure-core/src/main/java/io/seata/spring/boot/autoconfigure/properties/registry/RegistryNacosProperties.java:38]: 未涉及的逻辑和代码不要去改动

**Review by a364176773 — COMMENTED** [review]
yml中的使用方式需要补充进去

**Comment by hadoop835** [comment]
> yml中的使用方式需要补充进去

加在哪里呢

**Comment by a364176773** [comment]
> > yml中的使用方式需要补充进去
> 
> 加在哪里呢

server里的resources里的application.example.yml

**Pull request closed without merging**